firebase-firestore/ktx/src/androidTest/java/com/google/firebase/firestore/IntegrationTestUtil.kt

```diff
@@ -122,9 +119,20 @@ fun testDocument(): DocumentReference {
     return testCollection("test-collection").document()
 }
 
-inline fun <reified T : Exception> testAssertThrows(
-    crossinline runnable: () -> Any?
-): ThrowableSubject {
-    val exception: T = Assert.assertThrows(T::class.java) { runnable() }
-    return Truth.assertThat(exception)
-}
+
+/**
+ * Uncomment the code below to see the Compile time IDE Error: "Conflicting overloads Error".
+ *
+ * The reason for this error is I already have this method name defined in my unit test direcotry,
+ * since the package name: "package com.google.firebase.firestore" are the same, so IDE thought I
+ * used the same method name twice.
+ */
+
+
+
+// inline fun <reified T : Exception> assertThrows(
```

Comment:
I try to provide the `assertThrows` in the integration testutil, then I got an error that this method name is already used in the unit testutil.